mitodl/mit-open
# rename field to channel

### What are the relevant tickets?
Closes https://github.com/mitodl/mit-open/issues/734

### Description (What does it do?)
This PR renames `FieldChannel` to `Channel`, along with all related code mentioning "field" or "fields" to "channel" or "channels."

### How can this be tested?
Realistically, the entire app should be smoke tested but I'm not sure what else to recommend here as it's such a massive rename. First of all, when you spin up the app the migrations should apply successfully. Beyond that, the code should be gone through with care to ensure that there haven't been any unintended side effects.
